### ISSUE TITLE
error  Parsing error: [graphql-eslint]: Syntax Error: Unexpected Name "query"

### ISSUE THREAD

**Issue by AndysonDK** [dotansimha/graphql-eslint#809]
**Describe the bug**
When I add a block comment above a graphql query like this:
`users.graphql`
```Graphql
"""
lol
"""
query Users {
  users {
    id
    name
  }
}
```
I am receiving the following error:
<img width="533" alt="image" src="https://user-images.githubusercontent.com/28491857/143998195-865aa3e3-df24-4fcf-857f-638118a97371.png">

**To Reproduce**
Clone this [repo ](https://github.com/AndysonDK/syncbase) and run `yarn` in root folder to install all the required packages, then add the comment
```Graphql
"""
lol
"""
```
right above `query Users {` located in this [file](https://github.com/AndysonDK/syncbase/blob/main/packages/web/src/graphql/queries/users.graphql), then run `yarn eslint .` in root folder of the repo, the following error should appear:
<img width="500" alt="image" src="https://user-images.githubusercontent.com/28491857/143998435-8bea8d12-f3f5-4695-a488-5acb42f6b361.png">

**Expected behavior**
I would expect no errors to appear as I think I would be able to add a comment which by default is required to be `block` styled by `graphql-eslint`. It seems though that comments using `#` is working fine.

**Environment:**

- OS: WSL2 (Ubuntu 20.04.3 LTS)
- @graphql-eslint/eslint-plugin: "^3.0.1"
- NodeJS: v14.17.6
- [Eslint config file](https://github.com/AndysonDK/syncbase/blob/main/.eslintrc.js)

If it's by design that there shouldn't be any descriptions with `""" ... """` on graphql operations and only in schemas then fine, but the error should probably have a better error message then.

**Comment by B2o5T** [comment]
Hi @AndysonDK, you can't use `"""` comments in operations or fragments, for that reason graphql parser breaks.

Your [eslint config](https://github.com/AndysonDK/syncbase/blob/main/.eslintrc.js#L179) is wrong.

```js
      files: ["*.graphql"],
      extends: [
        "plugin:@graphql-eslint/schema-recommended",
        "plugin:@graphql-eslint/operations-recommended",
      ],
```

You can't run schema rules on operations files and operations rules on schema files.

Try to refactor your `overrides` section to something like this if you have monorepo project:

```js
{
  files: "PATH_TO_YOUR_SCHEMA_FILES/**/*.graphql",
  extends: "plugin:@graphql-eslint/schema-recommended",
  ...
},
{
  files: "PATH_TO_YOUR_OPERATIONS_FILES/**/*.graphql",
  extends: "plugin:@graphql-eslint/operation-recommended",
  ...
}
```

**Comment by AndysonDK** [comment]
I haven't seen that you should do that in any parts of the docs, my current configuration works well besides the `"""` on operations though
